### PR TITLE
Added UIA support for split container

### PR DESCRIPTION
**Pull request by Danil-Andrianov**
Partially implements #3421

## Proposed changes

- Updated "SupportsUiaProviders" flag.
- Added and implemented splitContainer accessible object
- Added unit tests

## Customer Impact

**Before update:**

InspectPropertiesWithoutUIA

AIPropertiesWithoutUIA

**After update:**

InspectPropertiesWithUIA

AIPropertiesWithUIA

## Regression? 

- No

## Risk

- Minimal

## Test methodology 

- CTI team

## Accessibility testing  

- Narrator
- Inspect
- Accessibility Insights

## Test environment(s) 

- Microsoft Windows [Version 10.0.22000.318]
- .NET Core SDK: 7.0.0-alpha.1.21562.1

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/6519)

### CODE REVIEW

**Comment by JeremyKuhne** [comment]
@Danil-Andrianov I'm having a hard time seeing what is different between the screenshots.

**Comment by Danil-Andrianov** [comment]
@JeremyKuhne the difference is just in provider, please, check screenshots from Inspect. AI data are same in both cases.

**Review by RussKie — APPROVED** [review]

**Review by dreddy-work — APPROVED** [review]

**Review by SergeySmirnov-Akvelon — DISMISSED** [review]
Looks good if it works

**Comment by Tanya-Solyanik** [comment]
@Olina-Zhang  - could you please test this change?

**Comment by Olina-Zhang** [comment]
Tested this PR for splitContainer control in Accessibility tools, it supports UIA provider and no issue was found finally.

**Comment by dreddy-work** [comment]
@Danil-Andrianov , can you please rebase this?

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/SplitContainer.SplitContainerAccessibleObjectTests.cs [hunk 1]
```diff
@@ -67,11 +67,55 @@ public void SplitContainerAccessibleObject_GetPropertyValue_ControlType_IsExpect
...
+        [InlineData((int)UIA.AutomationIdPropertyId, "SplitContainer1")]
+        public void SplitContainerAccessibleObject_GetPropertyValue_Invoke_ReturnsExpected(int propertyID, object expected)
+        {
+            using var control = new SplitContainer
+            {
+                Name = "SplitContainer1",
+                AccessibleName = "TestName"
+            };
+
+            var accessibleObject = new SplitContainer.SplitContainerAccessibleObject(control);
```
Comment (vladimir-krestov) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/SplitContainer.SplitContainerAccessibleObjectTests.cs:88]: Please use one style for all tests, as you did before:
```suggestion
            using SplitContainer control = new()
            {
                Name = "SplitContainer1",
                AccessibleName = "TestName"
            };

            SplitContainer.SplitContainerAccessibleObject accessibleObject = new(control);
```

**Review by vladimir-krestov — DISMISSED** [review]
Looks good ✔️

**Pull request merged**